.ruff.toml

```diff
@@ -0,0 +1,230 @@
+# Ruff configuration file adapted from astropy
+extend = "pyproject.toml"
+lint.ignore = [
```

Comment:
Are these global ignores?

Reply:
Yes, these ignores should apply globally.